prowler/lib/outputs/csv/csv.py

```diff
@@ -78,6 +78,9 @@ def transform(self, findings: List[Finding]) -> None:
                 finding_dict["DEPENDS_ON"] = unroll_list(finding.metadata.DependsOn)
                 finding_dict["RELATED_TO"] = unroll_list(finding.metadata.RelatedTo)
                 finding_dict["NOTES"] = finding.metadata.Notes
+                finding_dict["ADDITIONAL_URLS"] = unroll_list(
+                    finding.metadata.AdditionalURLs
+                )
```

Comment:
This needs to be the very last column, not to break automations reading CSV columns.